cs/di-extensions.texy

```diff
@@ -5,73 +5,115 @@ DI: Rozšíření pro kontejner
 
 /---code neon
 extensions:
-	blog: MyBlogExtension
+	blog: BlogExtension
 \---
 
-Každé rozšíření `Compiler`u musí dědit od [Nette\DI\CompilerExtension|api:] a může implementovat tři různé metody, které jsou postupně volány, během sestavování `Container`u.
+Každé rozšíření `Compiler`u musí dědit od [Nette\DI\CompilerExtension|api:] a může implementovat tři různé metody, které jsou postupně volány, během sestavování `Container`u:
```

Comment:
To `Compiler`u apod vypadá docela divně. Buď bych napsal *rozšíření třídy Compiler*, ale ve skutečnosti to je spíš *rozšíření kompileru*, protože to nutně nemusí být zrovna tato třída. 

Takže třeba: `Každé rozšíření kompileru dědí od [Nette\DI\CompilerExtension|api:] a může implementovat tři metody, které jsou postupně volány, během sestavování DI kontejneru:`